Makefile

```diff
@@ -109,6 +111,10 @@ test-with-cover:
 	$(MAKE) for-all CMD="go test -coverprofile=cover.out ./..."
 	$(MAKE) for-all CMD="go tool cover -html=cover.out -o cover.html"
 
+.PHONY: 
```

Comment:
Does the word `test-updater-integration` need to follow `.PHONY:` like it does in other commands?